### ISSUE TITLE
Google Lens hasn't returned any results for this query.

### ISSUE THREAD

**Issue by John1212121** [serpapi/public-roadmap#974]
I integrated the google_lens engine in my code. At the beginning I received roughly 3 responses with good content. But now, I always get as response "Google Lens hasn't returned any results for this query."

I guess this is a bug? Or do I do something wrong? 

**Details:** 
**This is my request code (dart):**

  final url = Uri.parse('https://serpapi.com/search');
  final apiKey =
      'xxxxx';
  final engine = 'google_lens';

  final response = await http.get(
    url.replace(queryParameters: {
      'api_key': apiKey,
      'engine': engine,
      'url': imageUrl,
    }),
  );

**This is the response I receive:**
{
    "search_metadata": {
xxxx
 },
    "search_parameters": {
        "engine": "google_lens",
        "url": "xxxx"
    },
    "search_information": {
        "images_results_state": "Fully empty"
    },
    "error": "Google Lens hasn't returned any results for this query."
}

**Comment by marm123** [comment]
Hi @John1212121! This looks like the issue with Google not returning any results for your image rather than anything with your implementation. If you contact us at contact@serpapi.com or through the chatbox widget at https://serpapi.com/ we can try to find the root cause of your issue.

**Comment by hartator** [comment]
@John1212121 Feel free to share more details here as well if you don't mind this being public.

**Comment by sonika-serpapi** [comment]
Another customer reached out regarding the same issue with some specific images on google lens. It is not returning results when our API is called with our Python library, but results are returned in our playground. 

Inspect: https://serpapi.com/searches/66e9dde7dda9d947646775bb/inspect 
Playground example: https://serpapi.com/playground?engine=google_lens&url=https%3A%2F%2Fapi.veritem.com%2Fdata%2Finput-images%2F2024-09-17-DQ1XK65-airpods.png
JSON response to user: https://serpapi.com/searches/42cbe4deaf4d85fb/66e9dde7dda9d947646775bb.json

It could be something to do with how certain URL characters are escaped in Python. 

[Intercom](https://app.intercom.com/a/inbox/yzck75jr/inbox/shared/all/conversation/102910000070381?view=List)

**Comment by hartator** [comment]
> Inspect: https://serpapi.com/searches/66e9dde7dda9d947646775bb/inspect

There is no valid results here.

**Comment by sonika-serpapi** [comment]
> There is no valid results here.

Yes, but intermittently, the playground was showing results for this same query. I removed the playground link from the comment on the user's request. Later, they reached out and mentioned they were able to resolve the issue on their end.

**Comment by schaferyan** [comment]
I'll close this issue since we haven't recieve more reports.
